src/components/challengeWidget/challengeWidget.css

```diff
@@ -47,3 +48,29 @@
   justify-content: space-between;
   border-radius: 50%;
 }
+span {
+  display: block;
+  height: 3rem;
+  width: 3rem;
+  line-height: 3rem;
+
+  -moz-border-radius: 30px;
+  border-radius: 30px;
+  justify-content: center;
+  align-items: center;
```

Comment:
These properties are only applicable for `display: flex;`, if you are using `display: block;`, they will just be ignored.